cmd/endpoint-ellipses_test.go

```diff
@@ -105,6 +124,12 @@ func TestGetSetIndexes(t *testing.T) {
 			[][]uint64{{4}},
 			true,
 		},
+		{
```

Comment:
could we add a failure test case where the set sizes doesn't match any of the valid set sizes like 4, 8, etc? That would double up as documentation for which values are accepted and which aren't accepted.

Reply:
sure will do @krisis